src/app/pages/send-crypto-asset/components/amount-field.tsx

```diff
@@ -19,24 +24,27 @@ export function AmountField({ symbol, rightInputOverlay }: AmountFieldProps) {
         justifyContent="center"
         width={`${maxInputContainerWidth}px`}
       >
-        <Flex flexBasis="50%">
-          <Box
-            as="input"
-            id={amountInputId}
-            caretColor={color('accent')}
-            flexGrow={1}
-            maxLength={10} // TODO: Replace with asset decimals + 3?
-            outline="0px solid transparent"
-            placeholder="0"
-            pr="base-tight"
-            textAlign="right"
-            width="100%"
-            wordWrap="normal"
-            {...field}
-          />
-          {/* TODO: Replace with asset symbol */}
-          <Text flexShrink={2} fontSize={inputFontSize} width="100%">
-            {symbol}
+        <Flex flexGrow={1} justifyContent="flex-end">
+          <Box textAlign="right" width={`${symbolTextWidth}px`}>
+            <Box
+              as="input"
+              id={amountInputId}
+              caretColor={color('accent')}
+              fontSize="48px"
+              maxLength={10} // TODO: Replace with asset decimals + 3?
+              outline="0px solid transparent"
+              placeholder="0"
+              pr="base-tight"
+              textAlign="right"
+              width="100%"
+              wordWrap="normal"
+              {...field}
+            />
+          </Box>
```

Comment:
Couldn't both these properties either be put on the inner box or the flex? 

Also, duplication of `textAlign` property, shouldn't the inner inherit?

Reply:
All this was needed to get it to scale correctly, but you can play around with it too if want ...maybe there is a better way. The `textAlign` isn't duplication from what I saw? It was needed to align the input field itself and the the text inside the input.

Reply:
The div around the input is what triggers the scaling.